kmsauth/__init__.py

```diff
@@ -57,6 +57,9 @@ def __init__(
             endpoint_url=None,
             token_cache_size=4096,
             stats=None,
+            max_pool_connections=100,
```

Comment:
we should make these defaults to match the boto defaults.  Then override as needed.

Reply:
https://botocore.amazonaws.com/v1/documentation/api/latest/reference/config.html

defaults
`max_pool_connections` = 10
`connect_timeout ` = 60 (seconds)
`read_timeout ` = 60 (seconds)